common/src/main/java/org/apache/drill/common/config/NestedConfig.java

```diff
@@ -111,6 +111,14 @@ public String getString(String path) {
     return c.getString(path);
   }
 
+  public String getString(String path, String defaultValue) {
```

Comment:
Please add JavaDoc.

Reply:
won't it look strange that class and all its methods don't have Javadocs and then BOOM -> one documented method?

Reply:
All other methods from this class implement methods from the interface with JavaDocs. So technically, it is a single method without JavaDoc...